emsdk.py

```diff
@@ -665,6 +665,11 @@ def download_file(url, dstpath, download_even_if_exists=False, filename_prefix='
         sys.stdout.flush()
   except Exception as e:
     print("Error downloading URL '" + url + "': " + str(e))
+    # Modified code to handle common "...[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: ..." error
+    # Error results in continuous issues being raised for same problem: #133 #136 #140 #176 #6275 #6548 #6723 #9036  ...
+    r = "<urlopen error \[SSL: CERTIFICATE_VERIFY_FAILED\] certificate verify failed: unable to get local issuer certificate \(_ssl\.c:(\d+)\)>"
```

Comment:
Perhaps this doesn't need to be so precise? Just searching for `SSL: CERTIFICATE_VERIFY_FAILED` may be good enough, and simpler. Can just do a python string check `pattern in str(e)` for that.

Reply:
All done. See updatedcode